src/backend/internal/migrations/20240316125809_user_profile.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE user_gender AS enum('MALE', 'FEMALE', 'OTHER');
+
+ALTER TABLE
+    users
+ADD
+    COLUMN location VARCHAR(255),
```

Comment:
If the `location` field is changed to `address`, don't forget to modify this file as well.